build/Makefile

```diff
@@ -23,9 +23,9 @@ SCX_INTERMEDIATE_DIR := $(SCX_DIR)/intermediate/$(BUILD_CONFIGURATION)
 DSC_TARGET_DIR := $(DSC_DIR)/release
 
 RUBY_DIR := $(BASE_DIR)/source/ext/ruby
+RUBY_VER := 2.3.0
```

Comment:
I don't know if you need comments here about base system requirements (we don't have that for other base system requirements here). But they definitely do need to be documented in the `Build-OMS-Agent-for-Linux` project.